### ISSUE TITLE
Minimize, maximize and close icons

### ISSUE THREAD

**Issue by LvDaniel** [AntonyCorbett/JwlMediaWin#26]
Hi Antony,

Is there a way to hide "minimize, maximize and close" icons while using JwlMediaWin tool?
This can be a bit distracting especially when playing songs or displaying light-collored images.
Thank you!

![image](https://user-images.githubusercontent.com/45002147/164994822-ef1e4c11-9fa8-4ff1-865d-7f0eadaff378.png)

**Comment by AntonyCorbett** [comment]
Sorry, I can't reproduce what you see.

**Comment by LvDaniel** [comment]
On the 2nd screen, after JwlMediaWin fixes the JWL media window, I get those controls buttons (minimize, maximize, close) in black (while displaying the year text they aren't visible). I'm seeing this at KH and at home PC as well. If I try to click them, nothing happens.
So this is not the normal behavior? Am I doing something wrong? I'm using JwlMediaWin 1.1.0.1 on Win 10, 20H2 version.

If I minimize and maximize the JWL media window, I see those buttons in white. I I click anywhere, they disappear (as become black - non active window I believe).

![image](https://user-images.githubusercontent.com/45002147/165318645-067ac7e7-6fb2-493a-8880-cda886bae3d7.png)

**Comment by AntonyCorbett** [comment]
Ah yes - if the window is active then the buttons are visible, but this is the same as when you restore the JWL media window (without using JwlMediaWin). Can you avoid focusing the Media window? I think that is the only solution.

**Comment by LvDaniel** [comment]
Well, when focusing the window the buttons are white, while not being focused they are black, so I can't avoid them at all.
When I'm not using JwlMediaWin, the buttons are not appearing regardless the media window is focused or not.

**Comment by AntonyCorbett** [comment]
Without JwlMediaWin, the buttons appear as soon as you double-click the JWL Media window title bar, thus allowing the window to operate stand-alone. This is in effect what JwlMediaWin does in order to prevent the window from minimizing at inopportune moments,

So, why do you need to focus the window?

**Comment by LvDaniel** [comment]
@AntonyCorbett you are correct. If I'm not using JwlMediaWin and I double-click the JWL Media window, I can see the buttons. And if I double-click again, they disappear.
However, while using JwlMediaWin I don't do anything (I just open JWL) and once the window is fixed the buttons are there. If I try to double click on JWL Media window nothing happens, the buttons are still there. I just want to get rid of them and I don't know how, as they appear by default.

**Comment by AntonyCorbett** [comment]
> I don't do anything (I just open JWL) and the buttons are there.

Understood. That's where our experiences differ. When I open JWL, the media window is displayed in its native form on the second monitor. After a second or two JwlMediaWin transforms the window. The buttons are there but since the media window is not focused, they are invisible; hence my question "why do you need to focus the window?"

BTW - also see #18 (last two comments)

**Comment by LvDaniel** [comment]
Thanks for pointing me to https://github.com/AntonyCorbett/JwlMediaWin/issues/18. To be honest I forgot the conclusion or I thought that something changed in the meantime.
So what I'm seeing (and you don't encounter this) is due to JWL version (after 12.2) or due to Win 10 version?
There are some changes I can make to my environment to make them invisible or I can't change this?
Thanks Antony for your patience and help, it's highly appreciated.

**Comment by mlo-glitch** [comment]
Was this ever resolved? Was looking around to see what can be done since we are also seeing this...
